### DISCUSSION TITLE
graphql/codegen/docs/guides/svelte

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/graphql-code-generator#9364] [Docs Discussions]
# graphql/codegen/docs/guides/svelte

Generate typed code for SvelteKit and Apollo. GraphQL Code Generator and the community provide typed code generation for Apollo, SvelteKit _native_, and other clients.

https://the-guild.dev/graphql/codegen/docs/guides/svelte

**Comment by parkerault** [comment]
FYI, the link in "For more advanced configuration, please refer to the KitQL documentation" is broken.

**Comment by o-az** [comment]
KitQL doesn't show anything graphql
